test/parse.js

```diff
@@ -427,6 +427,67 @@ describe('parse', function () {
         });
     });
 
+    it('param with import', function () {
+        var res = doctrine.parse(
+            [
+                '/**',
+                ' * @param {import("path/to/module")} arg - The description.',
+                ' */'
+            ].join('\n'), { unwrap: true });
+        res.tags.should.have.length(1);
+        res.tags[0].should.have.property('title', 'param');
+        res.tags[0].should.have.property('name', 'arg');
+        res.tags[0].should.have.property('description', 'The description.');
+        res.tags[0].should.have.property('type');
+        res.tags[0].type.should.eql({
+            type: 'NameExpression',
+            name: 'import("path/to/module")'
+        });
+    });
+
+    it('param with import plus name', function () {
+        var res = doctrine.parse(
+            [
+                '/**',
+                ' * @param {import("path/to/module").SomeType} arg - The description.',
```

Comment:
Should have a test with multiple dots too, eg `import("path/to/module").SomeContainer.SomeOther.SomeType`.